types/spectest/tests/consensusdata/invalid_sync_committee_block_validation.go

```diff
@@ -1,6 +1,23 @@
 package consensusdata
 
+import (
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
 // InvalidSyncCommitteeBlockValidation tests an invalid consensus data with sync committee block data.
-func InvalidSyncCommitteeBlockValidation() *SpecTest {
-	panic("implement")
+func InvalidSyncCommitteeBlockValidation() *ConsensusDataTest {
+
+	cd := types.ConsensusData{
+		Duty:    testingutils.TestingSyncCommitteeDuty,
+		Version: spec.DataVersionCapella,
+		DataSSZ: []byte{1},
+	}
+
+	return &ConsensusDataTest{
+		Name:          "invalid sync committee",
+		ConsensusData: cd,
+		ExpectedError: "could not unmarshal ssz: expected buffer of length 32 receiced 1",
```

Comment:
there is a typo in the error.. it is from our code?

Reply:
Not from our code